src/libCom/osi/os/WIN32/osdEvent.c

```diff
@@ -84,33 +84,45 @@ epicsShareFunc epicsEventStatus epicsEventWait ( epicsEventId pSem )
     }
 }
 
+extern HANDLE osdThreadGetTimer(void); /* from osdThread.c */
```

Comment:
In similar cases (recent posix PI mutex changes) definitions like this have gone into a local private header under `os/*/`.

Reply:
I did consider putting it in `osdThread.h` but this is included in `epicsThread.h` and so the data seemed to be getting leaked further. Is that OK i.e. people should not rely on functions in osd? it doesn't need to be `epicsShareExtern` so that would block DLL access. Or should i create a new private header file that is only used by `osdEvent.c` and `osdThread.c` ?

Reply:
I think the best way forward is to create eg. `os/WIN32/osdThreadPvt.h`.  Creating the file is all that is necessary.  Private headers don't need any Makefile change.

Of course, historically Base has been quite sloppy about putting "private" definitions in public (installed) headers.  Over time of course some of these definitions get used, complicating later maintenance.  We've been trying to improve this situation.  I admit that we're still sometimes slightly sloppy and wrap such definitions in an `#ifdef EPICS_PRIVATE_API`.